docs/consume-packages/Central-Package-Management.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Central Package Management
+description: Manage your dependencies in a central location and how you can get started with central package management.
+author: jondouglas
+ms.author: jodou
+ms.date: 2/25/2022
+ms.topic: conceptual
+---
+
+# Central Package Management
+
+Dependency management is a core feature of NuGet. Managing depdendencies for a single project can be easy. Managing dependencies for multi-project solutions can prove to be difficult as they start to scale in size and complexity. In situations where you manage common dependencies for many different projects, you can leverage NuGet's central package management features to do all of this from the ease of a single location.
+
+Historically, NuGet package dependencies have been managed in two main locations:
+
+- `packages.config` - An XML file used in older project types to maintain the list of packages referenced by the project.
+- `<PackageReference />` - An XML element used in new project types that manages NuGet dependencies directly within project files.
+
+Starting with [NuGet 7.0](..\release-notes\NuGet-7.0.md), you can centrally manage your dependencies in your projects with the addition of a `Directory.Packages.props` file.
```

Comment:
versioning; It's not really going to be nuget 7.0, but NuGet 6.4 that matches .NET 7.

Reply:
Given that we're adding the functionality in 6.2, any reason why we shouldn't announce the functionality in 6.2?

Reply:
Just placeholders. Will confirm with Jeff